src/db/migration/migration_map.rs

```diff
@@ -79,6 +79,8 @@ create_migrations!(
     "0.14.0" -> "0.15.0" @ MigrationVoid,
     "0.14.1" -> "0.15.0" @ MigrationVoid,
     "0.15.0" -> "0.15.1" @ MigrationVoid,
+    "0.15.0" -> "0.15.2" @ MigrationVoid,
```

Comment:
Shouldn't we go through `0.15.1` to make sure we test that path?
```suggestion
    "0.15.0" -> "0.15.1" @ MigrationVoid,
```

Reply:
Oh, I see, you added a short-cut. Hmm.

Reply:
It's wasteful, but yeah, we can.